addons/purchase_requisition/models/purchase.py

```diff
@@ -109,6 +109,18 @@ def write(self, vals):
 class PurchaseOrderLine(models.Model):
     _inherit = 'purchase.order.line'
 
+    @api.depends('product_id', 'date_order')
```

Comment:
No need to repeat the depends variables from the super

Reply:
Done!